include/fmt/base.h

```diff
@@ -8,13 +8,15 @@
 #ifndef FMT_BASE_H_
 #define FMT_BASE_H_
 
-#include <limits.h>  // CHAR_BIT
-#include <stdio.h>   // FILE
-#include <string.h>  // strlen
+#ifndef FMT_IMPORT_STD
+#  include <limits.h>  // CHAR_BIT
+#  include <stdio.h>   // FILE
+#  include <string.h>  // strlen
 
 // <cstddef> is also included transitively from <type_traits>.
-#include <cstddef>      // std::byte
-#include <type_traits>  // std::enable_if
+#  include <cstddef>      // std::byte
+#  include <type_traits>  // std::enable_if
+#endif
```

Comment:
I think we should move `import std` to make headers self-contained.

Reply:
Add `import std` to every file with std includes?

Reply:
Just base.h which is included from all other headers.